src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -16,6 +17,35 @@ const propTypes = {
 };
 
 export default class RecipientOverview extends React.Component {
```

Comment:
When possible, we should make these functional components and import `useState` from React.

Reply:
Done.